Website/theme.config.jsx

```diff
@@ -0,0 +1,58 @@
+import { useRouter } from "next/router";
+
+export default {
+  logo: (
+    <>
+      <img src="/Drifty.svg" alt="logo" width="30" height="30"></img>
+      &nbsp;&nbsp;
+      <span>
+        <strong>Drifty Docs</strong>
+      </span>
+    </>
+  ),
```

Comment:
@abhilashreddysh Can you change the design of the left side of the header like the design as seen on [GitHub blog website](https://github.blog/)? 
> [!NOTE]
> The arrow that comes on hovering over the logo, are only visible on wide screen devices. You can follow that style. It looks really good, doesn't it :smile:?
![image](https://github.com/user-attachments/assets/5c3a89d0-6608-4cce-b959-8d70da687733)